app/scripts/views/form.js

```diff
@@ -465,18 +465,30 @@ define(function (require, exports, module) {
       return message;
     },
 
+    reevaluateSubmissionEnabling: function (self){
+      self.enableSubmitIfValid();
+    },
+
+    submissionEnablingTimer: undefined,
+
     /**
      * Descendent views can override.
      *
      * Descendent views may want to override this to allow multiple form
      * submissions or to disable form submissions. Return false or a
      * promise that resolves to false to prevent form submission.
      *
+     * Descendent may dismiss timer with clearTimeout(this.submissionEnablingTimer)
+     *
      * @return {promise || boolean || none} Reture a promise if
      *   beforeSubmit is an asynchronous operation.
      */
     beforeSubmit: function () {
       this.disableForm();
+
+      // reevaluates enabling form submission 1 minute after submission attempt
+      var self = this;
+      this.submissionEnablingTimer = setTimeout(self.reevaluateSubmissionEnabling, 60000, self);
```

Comment:
Make 60000 a constant on the view. just like `VERIFICATION_POLL_IN_MS` in confirm.js